.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This file shouldn't be in the PR, can you add this to the .gitignore so it isn't source controlled and update?